internal/cloud/backend_runTasks.go

```diff
@@ -47,6 +65,31 @@ func summarizeTaskResults(taskResults []*tfe.TaskResult) *taskResultSummary {
 	}
 }
 
+func summarizePolicyEvaluationResults(policyEvaluations []*tfe.PolicyEvaluation) *policyEvaluationSummary {
+	var pendingCount, errCount, passedCount int
+	for _, policyEvaluation := range policyEvaluations {
+		if policyEvaluation.Status == "unreachable" {
```

Comment:
more a go style question:.  Would it be more typical to use a switch-case statement here instead of if-else blocks? I'm not sure of idomatic golang.